tests/standalone_tests/conftest.py

```diff
@@ -90,26 +92,35 @@
 CONTROLLER_PORT = 4500
 CONTROLLER_SMPC_PORT = 4501
 
-GLOBALNODE_CONFIG_FILE = "testglobalnode.toml"
-LOCALNODE1_CONFIG_FILE = "testlocalnode1.toml"
-LOCALNODE2_CONFIG_FILE = "testlocalnode2.toml"
-LOCALNODETMP_CONFIG_FILE = "testlocalnodetmp.toml"
-GLOBALNODE_SMPC_CONFIG_FILE = "smpc_globalnode.toml"
-LOCALNODE1_SMPC_CONFIG_FILE = "smpc_localnode1.toml"
-LOCALNODE2_SMPC_CONFIG_FILE = "smpc_localnode2.toml"
-CONTROLLER_CONFIG_FILE = "testcontroller.toml"
-CONTROLLER_SMPC_CONFIG_FILE = "test_smpc_controller.toml"
-CONTROLLER_LOCALNODES_CONFIG_FILE = "test_localnodes_addresses.json"
-CONTROLLER_SMPC_LOCALNODES_CONFIG_FILE = "test_smpc_localnodes_addresses.json"
+GLOBALNODE_CONFIG_FILE = "globalnode.toml"
+LOCALNODE1_CONFIG_FILE = "localnode1.toml"
+LOCALNODE2_CONFIG_FILE = "localnode2.toml"
+LOCALNODETMP_CONFIG_FILE = "localnodetmp.toml"
+CONTROLLER_CONFIG_FILE = "controller.toml"
+CONTROLLER_LOCALNODES_CONFIG_FILE = "localnodes_addresses.json"
 CONTROLLER_OUTPUT_FILE = "test_controller.out"
+if USE_EXTERNAL_SMPC_CLUSTER:
+    GLOBALNODE_SMPC_CONFIG_FILE = "external_smpc_globalnode.toml"
+    LOCALNODE1_SMPC_CONFIG_FILE = "external_smpc_localnode1.toml"
+    LOCALNODE2_SMPC_CONFIG_FILE = "external_smpc_localnode2.toml"
+    CONTROLLER_SMPC_CONFIG_FILE = "external_smpc_controller.toml"
+    SMPC_COORDINATOR_ADDRESS = "http://dl056.madgik.di.uoa.gr:12314"
+else:
+    GLOBALNODE_SMPC_CONFIG_FILE = "smpc_globalnode.toml"
+    LOCALNODE1_SMPC_CONFIG_FILE = "smpc_localnode1.toml"
+    LOCALNODE2_SMPC_CONFIG_FILE = "smpc_localnode2.toml"
+    CONTROLLER_SMPC_CONFIG_FILE = "smpc_controller.toml"
+    SMPC_COORDINATOR_ADDRESS = "http://172.17.0.1:12314"
+CONTROLLER_SMPC_LOCALNODES_CONFIG_FILE = "smpc_localnodes_addresses.json"
```

Comment:
smpc related properties, should probably go on a separate module since the whole smpc thing is supposed to be a special mode in which the system can operate. This goes for other modules as well that are not testing modules. I m just leaving that here..

Reply:
https://team-1617704806227.atlassian.net/browse/MIP-657